plugins/services/src/js/reducers/JSONConfigReducers.js

```diff
@@ -11,7 +12,15 @@ import {
 module.exports = {
   id: simpleReducer('id'),
   instances: simpleIntReducer('instances'),
-  container,
+  container: combineReducers({
```

Comment:
We probably need to re-consider this for when we introduce the multiple-container, but this is a good stepping stone